### PR TITLE
Add lint rules to objgraph

### PR DESCRIPTION
**Pull request by pcostell**

### CODE REVIEW

File: objgraph.py [hunk 1]
```diff
@@ -775,7 +794,7 @@ def _short_typename(obj):
 
 
 def _long_typename(obj):
-    objtype =_get_obj_type(obj)
+    objtype = _get_obj_type(obj)
```
Comment (mgedmin) [inline objgraph.py]: I missed this when I merged the PR that introduced `_get_obj_type()`!  That's an excellent example of why adding a style checker to the Travis test suite is a good thing.

File: tox.ini [hunk 2]
```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py26, py27, py32, py33, py34
+envlist = py26, py27, py32, py33, py34, flake, flake_docs
```
Comment (mgedmin) [inline tox.ini]: Please remove this bit.

File: tox.ini [hunk 3]
```diff
@@ -1,5 +1,5 @@
...
 
 [testenv]
-commands = python tests.py
+commands = python tests.py
```
Comment (mgedmin) [inline tox.ini]: Text files on Unix are supposed to have one trailing newline at the end of the last line in this file.  Please revert this bit also.

**Comment by mgedmin** [comment]
Looks good, except for the now-unnecessary tox.ini changes.

**Comment by pcostell** [comment]
Cool tox.ini stuff fixed.

**Comment by mgedmin** [comment]
Thank you!

**Pull request merged**